test/pretty-printer/ppTest.ml

```diff
@@ -11,14 +10,17 @@ let generator =
            let lowercase_id =
              string_size ~gen:(char_range 'a' 'z') (return 1)
            in
-           let regular_id id = Id.R.mk id in
-           let modal_id id = Id.M.mk (id ^ "'") in
+           let regular_id id = Mtt.Id.R.mk id in
+           let modal_id id = Mtt.Id.M.mk (id ^ "'") in
            match size with
            | 0 ->
                oneof
                  [
                    return Expr.unit;
-                   map (fun idr -> Expr.var_r (regular_id idr)) lowercase_id;
+                   (* won't work until expressions with free variables can be pretty-printed *)
+                   (* map
+                      (fun s -> Expr.VarR (Mtt.Id.R.mk s))
+                      (string_size ~gen:(char_range 'a' 'z') (return 1)); *)
```

Comment:
This looks like some merge conflict resolved incorrectly.

Reply:
Yeah, this is it. Fixed too